lib/puppet/type/scheduled_task.rb

```diff
@@ -174,11 +174,11 @@ def insync?(current)
     end
 
     def should_to_s(new_value=@should)
-      self.class.format_value_for_display(new_value)
+      super(new_value)
     end
 
     def is_to_s(current_value=@is)
-      self.class.format_value_for_display(current_value)
+      super(current_value)
```

Comment:
Ah right... I think this exists because there is confusion about whether the `is_to_s` method is called with `nil` or `absent` if the resource doesn't exist. I expect the `should_to_s` to never be called with `nil`, but the T&P API doesn't clearly define that. But that's an issue for another day.